drivers/gles3/shaders/tonemap.glsl

```diff
@@ -507,6 +501,12 @@ void main() {
 	color.rgb = apply_color_correction(color.rgb, color_correction);
 #endif
 
+#ifdef USE_DEBANDING
+	// For best results, debanding should be done before tonemapping.
+	// Otherwise, we're adding noise to an already-quantized image.
```

Comment:
Comment needs to be updated, it can be the same as in https://github.com/godotengine/godot/pull/66317